packages/client/tests/functional/_utils/setupTestSuiteClient.ts

```diff
@@ -1,13 +1,5 @@
-import { createClient as createLibSqlClient } from '@libsql/client'
-import { neonConfig, Pool as neonPool } from '@neondatabase/serverless'
-import { connect } from '@planetscale/database'
-import { PrismaLibSQL } from '@prisma/adapter-libsql'
-import { PrismaNeon } from '@prisma/adapter-neon'
-import { PrismaPg } from '@prisma/adapter-pg'
-import { PrismaPlanetScale } from '@prisma/adapter-planetscale'
```

Comment:
I think that isolating their loading is better as they could affect the regular tests that don't use them. After adding them here, I had seen an error in jest that pointed out that something tried to do an import after the tests were finished, so I have a hunch that something like this may be happening in the drivers. Also, lately I am seeing more frequent timeouts on unrelated tests, so I am trying to isolate the problem. All in all, I think better isolation is good.

Reply:
It looks like it fixed the timeouts after a few runs.

Reply:
Seems like we still have some of these new timeouts https://github.com/prisma/prisma/actions/runs/6400641262/job/17374539235?pr=21333#step:8:245

Reply:
I have another idea for investigating the timeouts, will open a separate PR later.